BASE/src/ServerTelemetryChannel/Implementation/TransmissionSender.cs

```diff
@@ -197,9 +208,11 @@ private async Task StartSending(Transmission transmission)
                 try
                 {
                     TelemetryChannelEventSource.Log.TransmissionSendStarted(acceptedTransmission.Id);
+                    acceptedTransmission.CredentialEnvelope = this.CredentialEnvelope;
```

Comment:
A test is needed to ensure, both accepted and rejected transmission gets CredentialEnvelope before transmission is sent to breeze. Code looks good, this new test will give 100% confidence.

Reply:
I don't understand your comment here. What scenarios do you want to see tested?

Reply:
Can't find where do set CredentialEnvelope on the rejected transmission? (it may be followed up separate PR). If we miss to add credential on some transmissins, it'll lead to hard to track down data losses.